trieste/models/gpflux/sampler.py

```diff
@@ -72,6 +72,7 @@ def __init__(self, sample_size: int, model: GPfluxPredictor):
             )
             for _ in range(len(self._model_gpflux.f_layers))
         ]
+        self._encode = lambda x: model.encode(x)
```

Comment:
out of curiosity, does the following work?

```suggestion
        self._encode = model.encode
```

Reply:
Good point. I think it does.